guide/src/conversions.md

```diff
@@ -2,27 +2,45 @@
 
 PyO3 provides some handy traits to convert between Python types and Rust types.
 
-## `.extract()`
+## `.extract()` and the `FromPyObject` trait
 
-The easiest way to convert a Python object to a Rust value is using `.extract()?`.
+The easiest way to convert a Python object to a Rust value is using
+`.extract()`.  It returns a `PyResult` with a type error if the conversion
+fails, so usually you will use something like
 
-## `ToPyObject` trait
+```
```

Comment:
```suggestion
```ignore
```

Reply:
Ah yes, the guide is tested as well :+1: